api/envoy/config/rbac/v3/rbac.proto

```diff
@@ -66,19 +66,28 @@ message RBAC {
 
   // Should we do safe-list or block-list style access control?
   enum Action {
-    // The policies grant access to principals. The rest is denied. This is safe-list style
+    // The policies grant access to principals. The rest are denied. This is safe-list style
     // access control. This is the default type.
     ALLOW = 0;
 
-    // The policies deny access to principals. The rest is allowed. This is block-list style
+    // The policies deny access to principals. The rest are allowed. This is block-list style
     // access control.
     DENY = 1;
+
+    // All requests are allowed. The filter sets the `envoy.log` dynamic metadata key to "yes" if
+    // at least one policy matches, and "no" otherwise.
+    LOG = 2;
   }
 
-  // The action to take if a policy matches. The request is allowed if and only if:
+  // The action to take if a policy matches.
+  // The request is allowed if and only if:
   //
-  //   * `action` is "ALLOWED" and at least one policy matches
+  //   * `action` is "ALLOW" and at least one policy matches
   //   * `action` is "DENY" and none of the policies match
+  //   * `action` is "LOG"
+  //
+  // If the action is "LOG" and at least one policy matches, the `envoy.log` dynamic metadata key
```

Comment:
Above there is no requirement that at least one policy matches to allow the request, so why is this here?

Reply:
The above comment specifies when a request should be allowed. This comment specifies when a request should be logged.